tests/Infrastructure.IntegrationTests/Secrets/DevelopmentSecretsServiceTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Codidact.Infrastructure.IntegrationTests.Secrets
+{
+    public class DevelopmentSecretsServiceTest
+    {
+        private readonly DevelopmentSecretsService _devSecretService;
+
+        private readonly Dictionary<string, string> InMemoryConfiguration = new Dictionary<string, string>
+            {
+                {"DefaultConnection", "LocalDb"},
+            };
+        public DevelopmentSecretsServiceTest()
+        {
+
+            var config = new ConfigurationBuilder()
+              .AddInMemoryCollection(InMemoryConfiguration).Build();
+            _devSecretService = new DevelopmentSecretsService(config);
+        }
```

Comment:
The formatting is a bit weird here.

Reply:
Need to remove extra newline and add extra line between prop and method. And chance you can suggest?